frontend/src/v5/ui/components/viewer/cards/tickets/sequencingProperty/sequencingProperty.component.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { DateTimePickerProps, DateTimePicker } from '@controls/inputs/datePicker/dateTimePicker.component';
+import SequencingIcon from '@assets/icons/outlined/sequence-outlined.svg';
+import CloseIcon from '@assets/icons/controls/clear_circle.svg';
+import { useEffect } from 'react';
+import dayjs from 'dayjs';
+import { SequencesActionsDispatchers } from '@/v5/services/actionsDispatchers';
+import { useFormContext } from 'react-hook-form';
+import { SEQUENCING_MODULE_END, SEQUENCING_MODULE_START } from '@/v5/store/tickets/tickets.helpers';
+import { IconContainer, Icons } from './sequencingProperty.styles';
+
+export const SequencingProperty = ({ onChange, onBlur, value, ...props }: DateTimePickerProps) => {
+	const { getValues } = useFormContext();
+
+	const openSequencesCard = () => SequencesActionsDispatchers.showSequenceDate(new Date(value));
+
+	const getDateTimeBoundary = (limit) => {
+		const limitVal = getValues(limit);
+		if (limit === props.name || !limitVal) return undefined;
+		return dayjs(new Date(limitVal));
+	};
+
+	const handleChange = (newValue) => onChange(newValue ? newValue?.toDate()?.getTime() : newValue);
+
+	useEffect(() => { onBlur(); }, [value]);
+
+	return (
+		<DateTimePicker
+			components={{
+				OpenPickerIcon: () => (value && (
+					<Icons onClick={(e) => e.stopPropagation()}>
+						<IconContainer onClick={openSequencesCard}>
+							<SequencingIcon />
+						</IconContainer>
+						<IconContainer onClick={() => handleChange(null)}>
+							<CloseIcon />
+						</IconContainer>
+					</Icons>
+				)),
+			}}
+			value={value}
+			// onChange is a required prop in DatePicker, however it is not needed as onAccept works better
+			// (onChange triggers at every change: year, minutes, hours, etc., onAccept only when a date is finally chosen)
+			onChange={() => true}
+			onAccept={handleChange}
+			minDateTime={getDateTimeBoundary(SEQUENCING_MODULE_START)}
+			maxDateTime={getDateTimeBoundary(SEQUENCING_MODULE_END)}
+			{...props}
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/2e99ce68-93a9-40c1-a6ce-8d6e2eaeba23)
A user with viewer permissions should be able to click the 'Go to sequence' button, right?

Reply:
Currently viewers can set a date in safetibase sequences but it errors.
![image](https://github.com/3drepo/3drepo.io/assets/62481250/02e40df3-0917-4965-ba0c-6626f4c838cf)
I know this is unrelated to this PR so I'll make an issue for it

Reply:
> A user with viewer permissions should be able to click the 'Go to sequence' button, right?

Yes can interact but cannot edit